R/condition_message_linter.R

```diff
@@ -0,0 +1,56 @@
+#' Block usage of paste() and paste0() with messaging functions using ...
+#'
+#' `stop(paste0(...))` is strictly redundant -- `stop(...)` is equivalent.
+#'   `stop(...)` is also preferable to `stop(paste(...))`. The same applies to
+#'   all default condition functions, i.e., [stop()], [warning()], [message()],
+#'   and [packageStartupMessage()].
+#'
+#' @evalRd rd_tags("stop_paste_linter")
```

Comment:
old name needs to be replaced here for proper docs.

Reply:
ah good catch. this might be an issue elsewhere too...

Reply:
Found two: expect_s4_class_linter and conjunct_expectation_linter
![image](https://user-images.githubusercontent.com/2363178/160472102-f6515246-0794-4cff-840a-3d497c5a850e.png)

Reply:
I'll add the other one to #1014